docs/schema/guidance/generating-statements.rst

```diff
@@ -0,0 +1,29 @@
+.. _generating-statements:
+
+Generating statements
+=====================
+
+Statements SHOULD be treated as immutable: once a Statement is published it SHOULD NOT be republished with the same statement identifier (``statementId``) and different property values. See :ref:`information-updates` for more information.
+
+Each Statement MUST have an entirely unique statement identifier. There MUST NOT be a possibility of collision between identifiers for different Statements. 
```

Comment:
```suggestion
Each Statement MUST have a unique statement identifier. There MUST NOT be a possibility of collision between identifiers for different Statements. 
```

'entirely unique' does add some emphasis but I don't think it's needed because the following sentence reinforces that message.